qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -200,6 +200,7 @@ def __init__(self, *, win_id, private, parent=None):
         self.tabindex = tabindex.TabIndex()
         self.keystring = keystring.KeyString()
         self.prog = progress.Progress(self)
+        self.text_widgets = []
```

Comment:
Since this isn't used from outside this class, it should be made "private"

```suggestion
        self._text_widgets = []
```